xmtp_id/src/associations/signature.rs

```diff
@@ -314,25 +320,32 @@ impl LegacyDelegatedSignature {
     ) -> Self {
         LegacyDelegatedSignature {
             legacy_key_signature,
-            signed_public_key_proto,
+            signed_public_key_proto: signed_public_key_proto,
         }
     }
+
+    // pub fn new_with_bytes(signature_text: String, signature_bytes: Vec<u8>) -> Self {
+    //     let signed_public_key_proto =
+    //         sign_with_legacy_key(signature_text, signature_bytes).unwrap();
+    // }
 }
 
 #[async_trait]
 impl Signature for LegacyDelegatedSignature {
     async fn recover_signer(&self) -> Result<MemberIdentifier, SignatureError> {
         // 1. Verify the RecoverableEcdsaSignature
-        let legacy_signer = self.legacy_key_signature.recover_signer().await?;
+        // let legacy_signer = self.legacy_key_signature.recover_signer().await?;
```

Comment:
We still need to verify the signature from the private key. The only validation this function does is check which wallet signed the public key. But we also need to make sure that the signature of the signature text is valid.

Reply:
#775 should be fixing this.